webpack.config-preprocessor-directives.js

```diff
@@ -27,13 +27,17 @@ const isPackaging = process.env.PACKAGING || "0";
 const nodeModuleRelativeUrl = (isPackaging === "1") ?
     "node_modules" : "../node_modules";
 
+const infoMdRelativeUrl = (isPackaging === '1') ?
+    "assets/md/information" : "dist/assets/md/information"
+
```

Comment:
After reviewing the current usage of build-time variables defined in `webpack.config-preprocessor-directives.js`, as well as other cases in the source code where we need to reference known (relative) paths, I feel that we should leave `webpack.config-preprocessor-directives.js` unchanged, and simply add the `[dist/]assets/md/information` logic in `src/renderer/components/settings/Information.tsx`, where we already test the `__PACKAGING__` condition.

Reply:
Je comprend, mais actuellement webpack fonctionne,
Cyrille, m'a expliqué de faire comme ça, hier,

Je ne vais pas faire et défaire ce que je fais à chaque fois.

Nous avons des deadline à respecter.

La PR est testée sur MAC/PC et elle fonctionne.

Peux-tu la merger quand même a moins qu'il y ai qqe chose de vraiment bloquant.

Merci bcp Daniel.

Reply:
English please on GitHub :)

Reply:
* As a peer code reviewer I have a duty to highlight potential issues (e.g. regression bugs, architectural inconsistencies, etc.). Although the proposed solution "works", I observed that the addition of the `__INFO_MD_RELATIVE_URL__` "preprocessor directives" is out of place, from an architectural perspective. The proposed design is inconsistent with other usage in the codebase, because `__INFO_MD_RELATIVE_URL__` is domain-specific (i.e. particular to the logic of the Markdown "information file"), as opposed to; let's say; `__NODE_MODULE_RELATIVE_URL__` which is applicable to other use cases.
* As the developer currently responsible for approving + merging Pull Requests, I will follow the advice of other team members, and if the advice is: "this code works, we are under pressure because of a deliverable deadline, please disregard design issues, and merge as-is", then I will do so. However I stand by the fact that sloppy design decisions (even small ones) are a bad signal for current and future project contributors.

In cases where the problem is too complex to fix in time for a milestone release, sure, we can add a "TODO" comment in the source code, and file a separate actionable issue so that we can come back to it at a later stage.
In this particular case, it would take as much time to add a TODO and file an issue, as it would to fix the design problem in the first place.

Reply:
Re. the process, @danielweck is right: as reviewer, his task is to avoid evolutions which lead to bad design. So @panaC has to follow his advice.
Re. this PR, the initial info associated with the proposed modification is weak, so I can't take any position on its content.
Each PR must strike a balance between advantages, review time and deadlines.
Our focus in this sprint is accessibility, therefore our time should be spent on tasks related to accessibilty

Reply:
Ok, you're all right, it's important to have a clean code base.
We are a team, you are the project manager: I prefer in this case that you give me a briefing before coding the solution of each issue

I will make the change.

Reply:
Thank you both for your feedback. Onwards and upwards! :)